README.md

```diff
@@ -197,6 +176,14 @@ N.B: Do not delete the public schema, if you do, you will need to recreate it us
 
 See [Docs](./docs/README.md).
 
+Higher level Architecture diagrams can be found in the [Social Care System Architecture](https://github.com/LBHackney-IT/social-care-architecture/) repository.
+
+The process and tooling for diagram creation is found [here](https://github.com/LBHackney-IT/social-care-architecture/blob/main/process.md).
+
+### C4 Component Diagram
```

Comment:
Should we move the diagram to the top of the README?
A good diagram is what I look for 1st and it will be consistent across our repos.

Reply:
Makes sense, will update! 👍